transport-classes-io_uring/src/main/java/io/netty/incubator/channel/uring/Native.java

```diff
@@ -59,14 +61,17 @@
                 PeerCredentials.class, java.io.FileDescriptor.class
         );
 
+        File tmpDir = PlatformDependent.tmpdir();
+        Path tmpFile = tmpDir.toPath().resolve("netty_io_uring.tmp");
         try {
             // First, try calling a side-effect free JNI method to see if the library was already
             // loaded by the application.
-            Native.createFile();
+            Native.createFile(tmpFile.toString());
```

Comment:
The comment says "side-effect free" but creating a file is not. How did we get here?

Reply:
Also, this returns a file descriptor that isn't closed.

Reply:
side-effect free in terms of JNI.